packages/web/src/app/doc-pages/components/list-doc/list-icon-ceg/list-icon-ceg.component.html

```diff
@@ -1,21 +1,21 @@
 <ol class="e-list e-list--icons">
   <li>
     <span class="e-list__icon">
-      <i class="e-icon e-icon e-icon--powermeter_ams e-icon--md" aria-hidden="true"></i>
+      <e-icon name="powermeterAms" size="md"></e-icon>
     </span>
     <strong>Slik fungerer strømmåleren:</strong>
     Gjelder etablering og montering av nye målere og ved bytte av eksisterende målere.
   </li>
   <li>
     <span class="e-list__icon">
-      <i class="e-icon e-icon e-icon--han-color e-icon--md" aria-hidden="true"></i>
+      <e-icon name="hanColor" size="md"></e-icon>
     </span>
     <strong>HAN-port:</strong>
     For å åpne HAN-porten kan du logge inn på Min side og bestille dette.
   </li>
   <li>
     <span class="e-list__icon">
-      <i class="e-icon e-icon e-icon--powermeter_old e-icon--md" aria-hidden="true"></i>
+      <e-icon name="powermeterOld" size="md"></e-icon>
```

Comment:
Is setting the size to `md` necessarily here due to the list?

Reply:
Good catches! I was pretty much just blindly copying (oops), these aren't needed!